docs/conf.py

```diff
@@ -76,6 +77,8 @@
 
 
 .. _official test suite: https://github.com/json-schema-org/JSON-Schema-Test-Suite
+
```

Comment:
No need for this, the click docs are written in Sphinx, so normal intersphinx references will work as above.